C/H5D/testfiles/18/h5ex_d_nbit.ddl

```diff
@@ -1,7 +1,7 @@
 HDF5 "h5ex_d_nbit.h5" {
 GROUP "/" {
    DATASET "DS1" {
-      DATATYPE  32-bit little-endian integer 16-bit precision
+      DATATYPE  32-bit little-endian integer
```

Comment:
Did 1.8 not get the changes to h5dump that was in the other branches? Do we want to make that change?

Reply:
I guess not, at least for 1.8.22.
Did you mean to make the change to h5dump in 1.8.23?

Reply:
Maybe it was the next 1.8 release. I probably was using current development - so may need to add version checks for the tests?